suite-native/app/src/navigation/RootStackNavigator.tsx

```diff
@@ -23,14 +25,17 @@ const RootStack = createNativeStackNavigator<RootStackParamList>();
 export const RootStackNavigator = () => {
     const isOnboardingFinished = useSelector(selectIsOnboardingFinished);
     const userHasAccounts = useSelector(selectUserHasAccounts);
+    const { isDeviceConnectEnabled } = useIsDeviceConnectEnabled();
 
     const getInitialRouteName = () => {
-        if (isOnboardingFinished && userHasAccounts) {
-            return RootStackRoutes.AppTabs;
-        }
-        if (isOnboardingFinished && !userHasAccounts) {
-            return RootStackRoutes.AccountsImport;
+        if (isOnboardingFinished) {
+            if (userHasAccounts) return RootStackRoutes.AppTabs;
+
+            return isDeviceConnectEnabled
```

Comment:
This is not correct though. We also have the onboarding as well. Do you want to have these conditions separately each on one line?

Reply:
It's exactly same as before in term of funcionality and it's easier to extend and read. I did omit `if(isOnboardingFinished)` because that line doesn't change so I did rewrite only content of that if statement.

Reply:
```tsx
if (isOnboardingFinished) {
  if (useHasAccounts) return RootStackRoutes.AppTabs;
  if (isDeviceConnectEnabled) return RootStackRoutes.ConnectDevice

  return RootStackRoutes.AccountsImport
}
```

Reply:
https://github.com/trezor/trezor-suite/pull/9450/commits/3511bf66fdd094823584c7fc2b254d5a5ba18fe8